components/api-server/src/client.rs

```diff
@@ -334,20 +435,24 @@ pin_project! {
     /// * `FileStream`: Streaming from file system storage.
     /// * `MongoStream`: Streaming from MongoDB storage.
```

Comment:
We need to update the docstring here:

```suggestion
    /// * `MongoStream`: Streaming from MongoDB storage.
    /// * `S3Stream`: Streaming from S3 storage.
```